src/wp-includes/comment-template.php

```diff
@@ -1740,6 +1740,8 @@ function comments_popup_link( $zero = false, $one = false, $more = false, $css_c
  *                              to addComment.moveForm(), and appended to the link URL as a hash value.
  *                              Default 'respond'.
  *     @type string $reply_text The text of the Reply link. Default 'Reply'.
+ *     @type string $reply_to_text The text template for "Reply to %s". Default 'Reply to %s'.
+ *     @type bool   $use_reply_to_text Whether to use reply_to_text as visible link text. Default false.
```

Comment:
If we add this new argument, I think the name `$show_reply_to_text` might fit its purpose better.